airflow/requirements.txt

```diff
@@ -1,13 +1,15 @@
-/tmp/airflow_provider_weaviate-1.0.0-py3-none-any.whl
+/tmp/airflow_provider_weaviate-0.0.1-py3-none-any.whl
 apache-airflow-providers-github==2.3.1
 apache-airflow-providers-slack==7.3.2
 weaviate-client==3.22.0
 streamlit==1.25.0
-html2text
+html2text==2020.1.16
```

Comment:
May I know why do we want to pin these versions? If this is what we want, should we pin all the versions?

Reply:
Feel free to pin all.